src/bci_build/package/ruby/README.md.j2

```diff
@@ -0,0 +1,66 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+## Description
+
+[Ruby](https://www.ruby-lang.org/) is a dynamic, reflective, object-oriented, general-purpose, open-source programming language. According to its authors, Ruby was influenced by Perl, Smalltalk, Eiffel, Ada, and Lisp. It supports multiple programming paradigms, including functional, object-oriented, and imperative. It also has a dynamic type system and automatic memory management.
+
+## Usage
+
+To deploy an application, install dependencies, copy the sources, and configure the application's main script:
+
+```Dockerfile
+FROM {{ image.pretty_reference }}
+
+# throw errors if Gemfile has been modified since Gemfile.lock
```

Comment:
Maybe something like this instead?
```suggestion
# make bundler throw an error if the Gemfile and Gemfile.lock are not in sync
```

Reply:
_throw errors_ is a colloquialism. Better: _display error messages_

Reply:
Adjusted.